server/app/durablejobs/jobs/FixApplicantDobDataPathJob.java

```diff
@@ -0,0 +1,56 @@
+package durablejobs.jobs;
+
+import com.google.common.base.Preconditions;
+import durablejobs.DurableJob;
+import io.ebean.DB;
+import io.ebean.Database;
+import java.time.LocalDate;
+import models.Applicant;
+import models.PersistedDurableJob;
+import services.WellKnownPaths;
+import services.applicant.ApplicantData;
+
+/**
+ * Finds applicant data objects with applicant_date_of_birth set to a number and changes it to a
+ * nest object.
+ */
+public final class FixApplicantDobDataPathJob extends DurableJob {
+  private final Database database;
+
+  private final PersistedDurableJob persistedDurableJob;
+
+  public FixApplicantDobDataPathJob(PersistedDurableJob persistedDurableJob) {
+    this.persistedDurableJob = Preconditions.checkNotNull(persistedDurableJob);
+    this.database = DB.getDefault();
+  }
+
+  @Override
+  public PersistedDurableJob getPersistedDurableJob() {
+    return persistedDurableJob;
+  }
+
+  @Override
+  public void run() {
+    String sql =
+        "select * from applicants where ((object #>>"
+            + " '{}')::jsonb)::json#>'{applicant,applicant_date_of_birth}' is not null";
```

Comment:
suggestion (non-blocking, if-minor): Could you add additional criteria to the `where` clause to reduce the number of applicants in the result set?

Reply:
What additional criteria do you recommend?

Reply:
should we filter out be certain dates? Say only applicantdata createTime<12/31/2023

Reply:
done! filtered by `applicant, applicant_date_of_birth is not null` AND `applicant, applicant_date_of_birth, date is null`